src/aiida/cmdline/commands/cmd_node.py

```diff
@@ -25,6 +26,46 @@ def verdi_node():
     """Inspect, create and manage nodes."""
 
 
+@verdi_node.command('list')
+@click.option('-e', '--entry-point', type=str, required=False)
+@click.option(
+    '--subclassing/--no-subclassing',
+    default=True,
+    help='Pass `--no-subclassing` to disable matching subclasses of the specified `--entry-point`.',
+)
+@options.PROJECT(
+    type=click.Choice(
+        ('id', 'uuid', 'node_type', 'process_type', 'label', 'description', 'ctime', 'mtime', 'attributes', 'extras')
+    ),
+    default=('id', 'uuid', 'node_type'),
+)
+@options.PAST_DAYS()
+@options.ORDER_BY()
+@options.ORDER_DIRECTION()
+@options.LIMIT()
```

Comment:
I think it will be helpful to have default value say `100` for the limit to avoid giving to many results?

Reply:
How would the user turn it off then, if they want to see all results?

Reply:
Look at linux command like `head` and `tail` they use:

```
  -n, --lines=[+]NUM       output the last NUM lines, instead of the last 10;
                             or use -n +NUM to output starting with line NUM
```

Reply:
so how would you use `head -n` or `tail -n` to show _all_ lines? Is it even possible? I think this is not possible, because that would essentially be `cat`, right?

Reply:
Ah, true, it has to be `cat` or `tail -n +1` which means read from the first line. But that seems over-complex for `verdi` case. I withdraw the suggestion.